bin/eslint-github-init.js

```diff
@@ -68,14 +71,51 @@ inquirer.prompt(questions).then(answers => {
     eslintrc.extends.push('plugin:github/browser')
   }
 
-  if (answers.flow) eslintrc.extends.push('plugin:github/flow')
+  if (answers.typeSystem === 'flow') eslintrc.extends.push('plugin:github/flow')
+  if (answers.typeSystem === 'typescript') {
+    eslintrc.extends.push('plugin:github/typescript')
+
+    // Create a `tsconfig.json` if one doesn't exits. Equivalent to running `tsc --init`.
+    const tsconfigPath = path.resolve(process.cwd(), 'tsconfig.json')
+    if (!fs.existsSync(tsconfigPath)) {
+      const tsconfigDefaults = {
+        compilerOptions: {
+          target: 'es5',
+          module: 'commonjs',
```

Comment:
Is this correct? We don't use `commonjs` for authoring or artefacts. Should this be `esnext`?

Reply:
From what I can read I guess we want `ESNext` as that supports `import()` as well as `import`